### PR TITLE
Treatment layout

### PR DESCRIPTION
**Pull request by pflopez**
- Adds breadcrumbs and main layout on treatment pages
- Adds menu on the titlebar
- Adds modals to confirm deletion and actions after duplicate plan

### CODE REVIEW

File: src/interface/src/app/treatments/after-duplicate-treatment-dialog/after-duplicate-treatment-dialog.component.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.padded {
+  padding: 0 16px;
+}
```
Comment (lastminutediorama) [inline src/interface/src/app/treatments/after-duplicate-treatment-dialog/after-duplicate-treatment-dialog.component.scss:3]: Should we (or I) somehow include this as part of the dialog component?

Comment (pflopez) [reply src/interface/src/app/treatments/after-duplicate-treatment-dialog/after-duplicate-treatment-dialog.component.scss:3]: yeah...i found myself repeating a bunch of stuff just to get a dialogs (like all this special dialog components, which ideally I should be using something more generic).

Comment (pflopez) [reply src/interface/src/app/treatments/after-duplicate-treatment-dialog/after-duplicate-treatment-dialog.component.scss:3]: But that would be outside the scope of this pr, i think we need some tlc on wrapping up the modals to be more reusable

**Review by lastminutediorama — APPROVED** [review]
lgtm

**Pull request merged**